website/src/pages/index.js

```diff
@@ -1,161 +1,151 @@
+
+          <p> Why scala-cli?
+            Scala-cli combines all the features you need to learn and use Scala in your (simple) projects.
+            Scala-cli is intuitive <code>and interactive</code>.
           </p>
           <p>
-            Should scala-cli also target simple libraries, so that we should provide support for cross compiling?
+            scala-cli is 
+            We believe that you can just just skip reading rest of this page and just <a href="docs/installation">install</a> and <a href="docs/input">enjoy scala-cli</a>:
           </p>
+
           <p>
-            Do you think that having a `scala-compose` or similar tool that will allow to combine multiple scala-cli projects is a good idea?
+            Still here?
+            <br/>
+            Ok, let us convince you why scala-cli is worth to give it a go.
           </p>
-          <p>Let us know!</p>
-        </p>,
 
-      image: "img/scala-spiral.png",
-      imageAlign: "right",
-    }
-  ];
-  return (
-    <div>
-      {features.map((feature) => (
-        <div className="hero text--center" key={feature.title}>
-          <div className={`container ${feature.imageAlign === 'right' ? 'flex-row' : 'flex-row-reverse'}`}>
-            <div className="padding--md">
-              <h2 className="hero__subtitle">{feature.title}</h2>
-              <p>{feature.content}</p>
-            </div>
-            <div className="padding-vert--md img-holder">
-              <img src={feature.image} />
-            </div>
+
+          <code>features marked as code are still under development</code>
+        </Section>
+
+        {/* Common features */}
+        <div className="container text--left row common_features justify-content-start">
+          <div className="col col--3 col--offset-1 red_border padding--md">
+            <h2>Intuitive, simple</h2>
+
+            <p>No complicated mechanism, tasks, plugins or extension. Single-module only </p>
+
+            <p>
+            All our commands have multiple aliases and follow well-known conventions.
+            </p>
+          </div>
+
+          <div className="col col--3 red_border padding--md">
+            <h2>Fast</h2>
+
+            <p>Scala-cli is optimize to respond as quickly as possible.</p>
+
+            <p>CLI is compiled to native code and compilations are offloaded to <a href="https://scalacenter.github.io/bloop/">bloop</a></p>
+          </div>
+
+          <div className="col col--3 red_border padding--md">
+              <h2>Command-line first</h2>
+
+              <p>Scala-cli does not require any configuration file and all in-file configuration can be overridden by command-line.</p>
+
+              <p>No additional installation or setup of enviroment (like specific CWD) required.</p>
```

Comment:
What do you mean by "specific CWD"?

Reply:
specific current working directory like whit sbt